# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/PresShell.h
# Commit: 4544c8e0eda1
# Full Hash: 4544c8e0eda1a54158ce635db02c449fa4e4338e
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2025-09-30 17:19 +0000
# Regressor Bug: 1987945
# File Overlap Count: 2
# Overlapping Files: accessible/base/nsCoreUtils.cpp, accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Bug 1987945 - P1: Implement details relationship for CSS anchored local accessibles. r=Jamie,emilio
# ==============================================================================

diff -r b8522d57b4e6 -r 4544c8e0eda1 layout/base/PresShell.h
--- a/layout/base/PresShell.h	Tue Sep 30 16:58:04 2025 +0000
+++ b/layout/base/PresShell.h	Tue Sep 30 17:19:31 2025 +0000
@@ -777,6 +777,10 @@
     mAnchorPosPositioned.RemoveElement(aFrame);
   }
 
+  const nsTArray<nsIFrame*>& GetAnchorPosPositioned() const {
+    return mAnchorPosPositioned;
+  }
+
 #ifdef MOZ_REFLOW_PERF
   void DumpReflows();
   void CountReflows(const char* aName, nsIFrame* aFrame);
